gets has been removed from recent versions of glibc, causing this call
of GL_WARN_ON_USE to fail. Simply remove it.

--- stdio.in.h.old	2013-03-20 15:01:34.794534692 +0100
+++ stdio.in.h	2013-03-20 15:01:50.424450017 +0100
@@ -712,10 +712,6 @@
 #  undef gets
 # endif
 _GL_CXXALIASWARN (gets);
-/* It is very rare that the developer ever has full control of stdin,
-   so any use of gets warrants an unconditional warning.  Assume it is
-   always declared, since it is required by C89.  */
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
 
